test/components/radio_test.rb

```diff
@@ -0,0 +1,230 @@
+require 'component_test_helper'
+
+class RadioTest < ComponentTestCase
+  def component_name
+    "radio"
+  end
+
+  test "does not render radio when no data is given" do
+    assert_empty render_component({})
+  end
+
+  test "does not render radio when no items but no name is given" do
```

Comment:
Not sure this test name is clear... Also can we tweak the name of this test and the previous one to indicate that the first is meant to render nothing and this one is meant to throw an error?